Packages/com.unity.inputsystem/InputSystem/Controls/InputControlLayout.cs

```diff
@@ -566,7 +595,19 @@ public class Builder
             /// </summary>
             /// <value>Name of base layout.</value>
             /// <seealso cref="InputControlLayout.baseLayouts"/>
-            public string extendsLayout { get; set; }
+            public string extendsLayout
+            {
+                get => m_ExtendsLayout;
+                set
+                {
+                    if (!string.IsNullOrEmpty(value))
```

Comment:
What if the user enters a load of whitespace? Is that ok or should we use IsNullOrWhiteSpace?

Reply:
I'd classify that under *what* you put in those strings is up to you as long as you put *something* in there :) The check is here mainly to filter out empty strings and make them come out as null. Should probably be `value != ""` instead of `IsNullOrEmpty`.